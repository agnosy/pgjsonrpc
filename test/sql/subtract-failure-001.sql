\set ECHO none
BEGIN;
\i sql/pgjsonrpc.sql
\set ECHO all

-- You should write your tests

SELECT jsonrpc.execute('
{
  "jsonrpc": "2.0",
  "method": "subtract",
  "params": [
    42,
    23
  ],
  "id": 1
}
');

ROLLBACK;
