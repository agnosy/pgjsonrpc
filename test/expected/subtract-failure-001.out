\set ECHO none
-- You should write your tests
SELECT jsonrpc.execute('
{
  "jsonrpc": "2.0",
  "method": "subtract",
  "params": [
    42,
    23
  ],
  "id": 1
}
');
                                           execute                                            
----------------------------------------------------------------------------------------------
 {"id" : "1", "jsonrpc" : "2.0", "error" : {"code" : -32601, "message" : "Method not found"}}
(1 row)

ROLLBACK;
